ophyd/epics_motor.py

```diff
@@ -129,6 +146,93 @@ def position(self):
         '''
         return self._position
 
+    @property
+    @raise_if_disconnected
+    def offset(self):
+        '''The current user offset value of the motor in its engineering units
+
+        Returns
+        -------
+        offset : float
+        '''
+        return self.user_offset.get()
+
+    @property
+    @raise_if_disconnected
+    def offset_frozen(self):
+        '''Whether or not the motor offset is frozen
+
+        Returns
+        -------
+        frozen : bool
+        '''
+        return bool(self.motor_offset_freeze_switch.get())
+
+    @property
+    @raise_if_disconnected
+    def at_high_limit_switch(self):
```

Comment:
Properties like this are convenient because they save the user from typing `.get()`, but they crowd the namespace. And they might confuse users who try to do `some_motor.offset = 5` and have to be told to do `some_motor.user_offset.put(5)` instead. I think I am against adding more of them, but I could be persuaded that I am wrong. Thoughts?